### PR TITLE
feat: 1641 add charge prices tab

### PR DESCRIPTION
**Pull request by ShaBloze**
## Description

Add "Prices" tab in the sidemenu which navigates to an empty page.
This will be feature flagged.

## References

- #1641
https://app.zenhub.com/workspaces/team-volt-60a61059028fd7000e49956e/issues/energinet-datahub/geh-charges/1641

### CODE REVIEW

File: libs/dh/charges/feature-prices/src/lib/dh-charges-prices.component.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnInit, NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'dh-dh-charges-prices',
```
Comment (HenrikSommer) [inline libs/dh/charges/feature-prices/src/lib/dh-charges-prices.component.ts:21]: It the double prefix "dh-dh-" here on purpose?

**Review by HenrikSommer — COMMENTED** [review]
LGTM, be aware of the two code smells identified by SonarCloud

File: libs/dh/charges/feature-prices/src/lib/dh-charges-prices.component.ts (hunk 1 above)
Comment (ShaBloze) [reply libs/dh/charges/feature-prices/src/lib/dh-charges-prices.component.ts:21]: That is a mistake. I removed the double prefix.

**Review by ManBearTM — CHANGES_REQUESTED** [review]
Code looks good, but you need to fix the linting errors. Consider installing the "eslint" extension in VS code if you haven't done so already (also check out the list of recommended extensions in VS code, just do `ctrl + shift + p` and type "Show Recommended Extensions").

**Review by ManBearTM — APPROVED** [review]

**Pull request merged**